pkg/pillar/cmd/nim/nim.go

```diff
@@ -34,10 +33,7 @@ import (
 )
 
 const (
-	agentName   = "nim"
-	tmpDirname  = "/var/tmp/zededa"
-	DNCDirname  = tmpDirname + "/DeviceNetworkConfig"
-	DPCOverride = tmpDirname + "/DevicePortConfig/override.json"
```

Comment:
So, do we not support DPC configuration override after this?

Reply:
We, but the code isn't using the specific filename any more, nor do we use DeviceNetworkConfig for override. We use a DevicePortConfig for override.